src/Features/Core/Portable/Completion/Providers/AbstractRecommendationServiceBasedCompletionProvider.cs

```diff
@@ -93,13 +108,18 @@ private static bool IsValidForTaskLikeTypeOnlyContext(ISymbol symbol, TSyntaxCon
             }
 
             return namedType.IsAwaitableNonDynamic(context.SemanticModel, context.Position) ||
-                   namedType.GetTypeMembers().Any(static (m, context) => IsValidForTaskLikeTypeOnlyContext(m, context), context);
+                   namedType.GetTypeMembers().Any(IsValidForTaskLikeTypeOnlyContext, context);
```

Comment:
Please revert. The reason for static here was to make it clear this was not allocating. Thanks.

Reply:
Method group approach doesn't allocate here as well, right? So `static` is only for explicitness here?

Reply:
Correct. It's about clarity here.